README.rst

```diff
@@ -40,43 +40,49 @@ Installation
 Install ticketutil with ``pip install ticketutil``.
 
 ticketutil is compatible with Python 2.7, 3.3, 3.4, 3.5, and 3.6.
-Note: For Python 2.6 and lower, an additional package, importlib, may 
+Note: For Python 2.6 and lower, an additional package, importlib, may
 need to be installed.
 
-If not installing with pip, a short list of packages defined in the 
-requirements.txt file need to be installed. To install the required 
+If not installing with pip, a short list of packages defined in the
+requirements.txt file need to be installed. To install the required
 packages, type ``pip install -r requirements.txt``.
 
 Usage
 -----
 
+Note: To enable debug logging for ticketutil, set an environment
+variable named TICKETUTIL_DEBUG to 'True'. If this environment variable
+is set to anything else or does not exist, debug logging will be
+disabled.
+
```

Comment:
^ This should be merged with "Note on the logging" below. I would prefer this to be removed and you to use original. IMHO this even is the case as I can not find any mention of `TICKETUTIL_DEBUG`:

```
(branch: add_verify_project) $ grep TICKETUTIL_ ticketutil/*.py
ticketutil/ticket.py:LOG_LEVEL = os.environ.get('TICKETUTIL_LOG_LEVEL', 'INFO')
```